src/Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -24,6 +25,7 @@ public class SkiaEncoder : IImageEncoder
     private readonly ILogger<SkiaEncoder> _logger;
     private readonly IApplicationPaths _appPaths;
     private static readonly SKImageFilter _imageFilter;
+    private static readonly List<SKTypeface> _typefaces;
```

Comment:
Any reason this is a List instead of an array?

Reply:
No, I just picked it with early draft but now it can be an array as it is now intended to be readonly